README.md

```diff
@@ -25,6 +25,8 @@ Simple utility CLI for importing data into SwaggerHub Explore.
 >  `export-spaces`                 Export SwaggerHub Explore spaces to filesystem
 >
 >  `import-spaces`                 Import SwaggerHub Explore spaces from a file
+>
+>  `import-postman-collection`     Import Postman Collection from a file into SwaggerHub Explore
```

Comment:
worth adding the version here? `v2.1 Postman Collection`

Reply:
added